spec/token/common/ERC2981.behavior.ts

```diff
@@ -0,0 +1,67 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { describeFilter } from '@solidstate/library';
+import { ERC2981Mock } from '@solidstate/typechain-types';
+import { expect } from 'chai';
+import { BigNumber } from 'ethers';
+import { ethers } from 'hardhat';
+
+export function describeBehaviorOfERC2981(
+  deploy: () => Promise<ERC2981Mock>,
+  skips?: string[],
+) {
+  const describe = describeFilter(skips);
+
+  describe('::ERC2981', function () {
+    let tokenIdOne = BigNumber.from(1);
+    let tokenIdTwo = BigNumber.from(2);
+    let tokenIdThree = BigNumber.from(3);
+
+    let receiver: SignerWithAddress;
+    let instance: ERC2981Mock;
+
+    before(async function () {
+      receiver = (await ethers.getSigners())[1];
+      instance = await deploy();
+    });
+
```

Comment:
Must call `describeBehaviorOfERC165` with the relevant `interfaceId` for the EIP.  See `ERC721Base.behavior.ts` for an example.

Reply:
https://github.com/solidstate-network/solidstate-solidity/pull/147/commits/aa82c30bcbc867c17580076ba267737c027aecc5
https://github.com/solidstate-network/solidstate-solidity/pull/147/commits/e1c0b3ce994f0cca4d46c4c349c335d7d7bc57e9